dbt_subprojects/solana/models/_sector/stablecoins/stablecoins_solana_balances.sql

```diff
@@ -0,0 +1,59 @@
+ {{
+  config(
+        schema = 'stablecoins_solana',
+        alias = 'balances',
+        materialized='incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['token_mint_address', 'token_balance_owner', 'day'],
+        post_hook='{{ expose_spells(\'["solana"]\',
+                                    "sector",
+                                    "stablecoins",
+                                    \'["synthquest"]\') }}')
+}}
+
+with
+      stablecoin_tokens as (
+            select
+                  blockchain, symbol, address, decimals
+            from 
+                  {{ source('tokens_solana', 'stablecoins')}}
+)
+
+, updated_balances as (
+            select 
+                  day
+                  , bal.token_balance_owner
+                  , sd.symbol
+                  , sum(coalesce(token_balance,0)) as token_balance 
+                  , token_mint_address
+            from {{ ref('solana_utils_daily_balances') }}
+            inner join (select address, symbol from stable_definitions) sd on sd.address = bal.token_mint_address
```

Comment:
```suggestion
            inner join (select address, symbol from stablecoin_tokens) sd on sd.address = bal.token_mint_address
```

Reply:
ah, i think this is it, CTE name mismatch. i knew i had seen that error before, couldn't recall exactly what. need to document this!